components/filter/filter.js

```diff
@@ -505,6 +517,56 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 			listLabel = dimension.headerText;
 		}
 
+		let selectedListItems = nothing;
+		let listItems = nothing;
+		if (dimension.selectedFirst) {
+			if (listLabel) listLabel = this.localize('components.filter.selectedFirstListLabel', { headerText: dimension.headerText });
```

Comment:
If there's no header text, do we need to tell users the selected items are first?

Reply:
I don't think so. The reason we are adding that information to the label is specifically to differentiate the selected items from the items below the header text since the selected items are displayed above the header text and may be unrelated.

If there is no separation of the list, then I don't think that clarification needs to be made and the user would be able to find out which items are selected when navigating the list.